src/pages/CreatePool/index.tsx

```diff
@@ -361,46 +365,48 @@ export default function CreatePool(): React.ReactElement {
                 <Typography mb={2}>
                   {t("createPoolTypeDescription3")}
                 </Typography>
-                <ToggleButtonGroup
-                  value={poolType}
-                  color="secondary"
-                  size="large"
-                  exclusive
-                  onChange={(event, value: PoolType) => {
-                    if (value) {
-                      setPoolType(value)
-                      if (value === PoolType.Base) {
-                        setTokenInputs([tokenInputs[0], ""])
+                <FormControl fullWidth>
+                  <InputLabel id="pool-type-label">Choose Pool Type</InputLabel>
+                  <Select
+                    label="Choose Pool Type"
+                    id="pool-select"
+                    onChange={(e) => {
+                      if (e.target.value === PoolType.Base) {
+                        setPoolType(e.target.value as PoolType)
+                        setTokenInputs(["", ""])
                       } else {
-                        setTokenInputs([tokenInputs[0]])
+                        const pool =
+                          expandedPools.data.byAddress[e.target.value as string]
+                        if (!pool) {
+                          console.error("Unable to locate pool")
+                          return
+                        }
+                        const tokensLength = pool.tokens.length
+                        const lpTokenAddr = pool.lpToken.address
+                        setPoolType(pool.poolName as PoolType)
+                        setSelectedTokensLength(tokensLength)
+                        setTokenInputs([
+                          ...(Array(maxAmountOfTokens - tokensLength).fill(
+                            "",
+                            0,
+                          ) as string[]),
+                        ])
+                        setMetapoolBasepoolAddr(e.target.value as string)
```

Comment:
We are giving poolAddress here. But I think we need to give lp token address in here to display lp token addrss on the last textfield.
![Screen Shot 2022-11-03 at 10 42 10 PM](https://user-images.githubusercontent.com/75422800/199829872-8f3fc4ba-d5f7-43c9-bad5-1a7c581328cb.png)